liveblog.php

```diff
@@ -838,7 +838,7 @@ public static function get_entries_paged( $page, $last_known_entry = false, $id
 			//If no page is passed but entry id is, we search for the correct page.
 			if ( false === $page && false !== $id ) {
 				$index = array_search( (int) $id, array_keys( $entries ), true );
-				$index = $index++;
+				$index++;
```

Comment:
While the change probably makes sense, I don't think this is something that PHPCS picks up as a violation, is it? (Happy to be shown I'm wrong!)

Reply:
PHPCS flagged `$index = $index + 1`. I half-fixed it. Was finishing the fix.